xDripTests/Scenes/Root/Root/RootPresenterTests.swift

```diff
@@ -72,7 +72,8 @@ final class RootPresenterTests: XCTestCase {
         // Given
         let spy = RootDisplayLogicSpy()
         sut.viewController = spy
-        let response = Root.ShowAddEntryOptionsList.Response()
+        
+        let response = Root.ShowAddEntryOptionsList.Response(types: [])
```

Comment:
You shouldn't just test whether it's called or not. You should test how the data is formatted. Pass something there and make sure you get the correct output.